### ISSUE TITLE
Loop commands

### ISSUE THREAD

**Issue by evaera** [evaera/Cmdr#77]
`until`, `while`, maybe `for`

**Comment by redrubyyy611** [comment]
Any thoughts on this recently? (:

would be amazing if I could do a bind to run a given command on a loop and/or with a list of values to iterate through

**Comment by randomdudeinternet** [comment]
i agree that this should be done but that should get rate limited to if you aren't one of the higher ranks by default since if abused this could easily even with a rate limit allow for lagging servers as a completely new player

**Comment by redrubyyy611** [comment]
yes, I agree the `for` statement should be the first one to release/probably easier to code
i think of it...maybe something like this...

> `$RedRubyyy611@Cmdr foreach <array_var> <cmd>`

where:
- `<array_var>` must be an array
- `<cmd>` must be an already defined command (that isn't `foreach` itself)

and the `foreach` will take each value in the `array_var` and pass it to `<cmd>`


maybe initially limited to passing 1 value at a time
